camera_info:
  camera_matrix:
    data: [751.748554, 0.000000, 656.307374,
           0.000000, 754.534212, 368.645691,
           0.000000, 0.000000, 1.000000]
    rows: 3
    cols: 3
  
  distortion_coefficients:
    data: [-0.351268, 0.087101, 0.000186, -0.003012, 0.000000]
    rows: 1
    cols: 5
  
  rectification_matrix:
    data: [1.000000, 0.000000, 0.000000,
           0.000000, 1.000000, 0.000000,
           0.000000, 0.000000, 1.000000]
    rows: 3
    cols: 3
  
  projection_matrix:
    data: [494.303589, 0.000000, 647.948443, 0.000000,
           0.000000, 685.485107, 370.209981, 0.000000,
           0.000000, 0.000000, 1.000000, 0.000000]
    rows: 3
    cols: 4 